src/App.test.tsx

```diff
@@ -30,6 +30,4 @@ test("renders register page", async () => {
 
   render(<App />)
   await waitFor(() => promise)
-
```

Comment:
bitte nicht die einzige assertion im test löschen

Reply:
fand keine weiter, hast du eine Tipp? Weil, "Wellcome to ..." nicht mehr dort steht.

Reply:
Momentan wäre es /LOGIN/

Wobei wir machen wir hier kein Login, wir erstellen einen player -> /create player/

Reply:
Ich verstehe nicht, was und eine assertion nützt, welche keine Vernünftige Prüfung durchführt?